src/panels/NotesPanel.jsx

```diff
@@ -86,38 +94,7 @@ export default class NotesPanel extends Component {
     }
 
     updateContextTrayItemWithSelectedPickListOptions = (selectedPickListOptions) => {
```

Comment:
[DL2]  Can probably rename this to `updateSelectedPickListOptions` since it's no longer updating context tray item in this function.